xmppserver/src/main/java/org/jivesoftware/openfire/muc/MUCRoom.java

```diff
@@ -1308,6 +1308,48 @@ public void removeOccupant(@Nonnull final MUCOccupant occupant) {
         MUCEventDispatcher.occupantLeft(occupant.getOccupantJID(), occupant.getUserAddress(), occupant.getNickname());
     }
 
+    /**
+     * Clears the chat history of a room. Sends out message retraction stanzas to all occupants, for all the message in the chat history.
```

Comment:
Please describe in another term what 'clear' means, as it is easy to misinterpret this (eg: remove from memory vs revocation vs completely deleted from database).

Reply:
This is done now.